plugins/other/pacman

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# DESCRIPTION
+#
+# This plugin monitors the number of pending updates in pacman.
+#
+# WARNINGS
+#
+# This plugin will issue a warning state whenever any package can be updated.
+#
+# ENVIRONMENT
+#
+# env.WARNING The number of updates at which a warning will be issued. If not
```

Comment:
Novice users will appreciate the documentation of the plugin configuration with an example, e.g.:
```
The number of upgradeable packages that would trigger a warning can be configured:

[pacman]
env.WARNING 3
```